test/arrays.js

```diff
@@ -1,6 +1,7 @@
 (function() {
+  var _ = typeof require == 'function' ? require('../underscore') : window._;
```

Comment:
Fixed by changing the cli to one that's just a wrapper for the official
qunit
On Feb 8, 2015 1:26 PM, "John-David Dalton" <notifications@github.com>
wrote:

> In test/arrays.js
> <https://github.com/jashkenas/underscore/pull/2033#discussion_r24300854>:
>
> > @@ -1,6 +1,7 @@
> >  (function() {
> > +  var _ = typeof require == 'function' ? require('../underscore') : window._;
>
> Was there an option to prevent that or did they stop in a newer version?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jashkenas/underscore/pull/2033/files#r24300854>.
>